src/functions/__tests__/alphabetical.test.ts

```diff
@@ -88,4 +101,34 @@ describe('alphabetical', () => {
       expect(runAlphabetical([100, 1], 'a')).toEqual([]);
     });
   });
+
+  test('is able to trap object again', () => {
+    const document = new Document(
+      `'404':
+'200':`,
```

Comment:
@P0lip Can't this be put in a single line with a `\n`?

Reply:
Yup!